.github/workflows/build-rpm.yaml

```diff
@@ -38,11 +37,8 @@ jobs:
         run: |
           cd rpmbuild/BUILD
           if [[ ${{ github.ref_type }} = 'tag' ]]; then
-            # In case is a tag, get the tag value
-            GITHUB_REF=${{ github.ref }}
-            TAG_NAME=${GITHUB_REF#refs/tags/}
-            # Check if the tag matches v<x>.<y>.<z>(-[0-9A-Za-z-]+)?
-            if [[ ${TAG_NAME} =~ ^v([0-9]+\.[0-9]+\.[0-9]+)(-[[:alnum:]-]+)?$ ]]; then
+            # In case is a tag, check if the tag matches v<x>.<y>.<z>(-[0-9A-Za-z-]+)?
+            if [[ ${GITHUB_REF_NAME} =~ ^v([0-9]+\.[0-9]+\.[0-9]+)(-[[:alnum:]-]+)?$ ]]; then
```

Comment:
I'm not convinced about the regex. For example, the string "v1.2.3-4----" matches, but do we want to accept it?